src/modules/email.js

```diff
@@ -32,14 +39,19 @@ export default class Email {
         Body: {
           Text: {
             Charset: 'UTF-8',
-            Data: `Your activation code is ${code}`
+            Data: `Please go to ${url} to confirm your account.`
           }
         }
       },
       Source: this.fromAddress
     };
   }
 
+  /**
+   * Send an email message.
+   *
+   * @param {object} message Message object (see AWS SDK)
+   */
   async sendMessage(message) {
     try {
```

Comment:
May be worth putting a check here for `environment !== 'test'` and putting the URL in the log instead of sending an email. (Maybe a future revision if needed)

Reply:
Yep, on it!